src/avfoundation.cs

```diff
@@ -1931,6 +1931,10 @@ interface AVAudioSessionInterruptionEventArgs {
 
 		[Export ("AVAudioSessionInterruptionOptionKey")]
 		AVAudioSessionInterruptionOptions Option { get; }
+
+		[iOS (10, 3), NoMac, NoTV, NoWatch]
+		[Export ("AVAudioSessionInterruptionWasSuspendedKey")]
+		bool WasSuspended { get; }
```

Comment:
_Value is a boolean NSNumber,_ ^ that `bool` won't work without `[BindAs]` (which is not in `cycle9`)